### PR TITLE
Optimization/short preprunning and memory

### PR DESCRIPTION
**Pull request by eskombro**
Int -> Int16
Short list for preprunning
Small heuristic modifications looking for a win

### CODE REVIEW

File: internal/brain/suggestor.go [hunk 1]
```diff
@@ -56,17 +57,18 @@ func addNewLayerPrePruning(poss []int, node *tr.Node, playerId int) {
 		capturePairs(m, captureDirections, &new.Tab)
 		new.Value = getHeuristicValue(playerId, &new.Tab, &new.Captured)
 		newMovesToTest = append(newMovesToTest, &new)
+		if len(newMovesToTest) > maxTestingMoves {
```
Comment (ekelen) [inline internal/brain/suggestor.go:60]: 👍

File: internal/brain/pruning.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func MinimaxRecursivePruning(node *tr.Node, depth int16, max bool, alpha int16,
 		if node.Value >= WIN_SCORE {
 			node.WinMove = true
 		}
-		return node.Value
+		return node.Value - 2*node.Value/int16(totalDepth-depth)
```
Comment (ekelen) [inline internal/brain/pruning.go:69]: ⚡️

**Review by ekelen — APPROVED** [review]
👊 🌋

**Comment by eskombro** [comment]
👊👊👊

**Pull request merged**